src/micro_http/src/common/headers.rs

```diff
@@ -291,11 +314,12 @@ mod tests {
     use super::*;
 
     impl Headers {
-        pub fn new(content_length: i32, expect: bool, chunked: bool) -> Self {
+        pub fn new(content_length: i32, expect: bool, chunked: bool, accepted: MediaType) -> Self {
```

Comment:
/s/accepted/accept

Most of the tests don't use `accept`. How about we leave this constructor as it was before and when we want to test the accept header as well, we just set it?

Reply:
Done.